### PR TITLE
test improvements

### PR DESCRIPTION
**Pull request by garlick**
This PR improves the `t0002-basic.t` sharness test, to more comprehensively check for expected `PMIx_Get()` results for pre-set attributes, and splits out the barrier and bizcard tests to a new script that will be expanded as the fence and dmodex callbacks are filled in.

### CODE REVIEW

File: t/t0002-basic.t [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-test_description='Basic plugin tests on a single shell'
+test_description='Basic plugin tests'
```
Comment (grondo) [inline t/t0002-basic.t:3]: commit message typo: s/thoroghly/thoroughly/

File: t/src/getkey.c [hunk 2]
```diff
@@ -30,32 +30,42 @@ static struct optparse_option opts[] = {
     { .name = "rank", .has_arg = 1, .arginfo = "RANK",
       .usage = "Perform the getkey only on RANK (default=all)",
     },
+    { .name = "label-io", .has_arg = 0,
```
Comment (grondo) [inline t/src/getkey.c:33]: commit message typo: it's:arrow_right:its

File: t/t0003-bizcard.t [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+test_description='Basic plugin tests'
```
Comment (grondo) [inline t/t0003-bizcard.t:3]: Should `test_description` get an update?

**Review by grondo — APPROVED** [review]
LGTM! Just a few trivial comments so I've approved.

Maybe premature, but if you'd like I could add a chain-lint builder to the ci. Also would coverage be of any use?

**Comment by garlick** [comment]
> LGTM! Just a few trivial comments so I've approved.

Thanks, I'll fix those before setting MWP.

> Maybe premature, but if you'd like I could add a chain-lint builder to the ci. Also would coverage be of any use?

I would welcome either.  I was just moving fast when I set things up and didn't want to pause for that.  However, now I regret it a bit, especially coverage.

**Pull request merged**